### PR TITLE
Top categories

### PR DESCRIPTION
**Pull request by bakozetra**
Hello Team , 

I have finished the component TopCategories. Please have a look at it, feel free to add comments  if necessary . 

Refs : #45e54j 

Thanks in advance

### CODE REVIEW

**Review by vakodrazan — APPROVED** [review]

**Review by ravinalamada — APPROVED** [review]

File: cypress/integration/TopCategories.spec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+
+describe('Storybook', () => {
+    beforeEach('visits storybook', () => {
+        cy.visit('http://localhost:6006')
+        cy.contains('TopCategories', { timeout: 20000 })
+    })
+    it('Has the unChecked', () => {
```
Comment (andrzej-onja) [inline cypress/integration/TopCategories.spec.js]: `Has the unChecked` ???

File: src/components/TopCategories/TopCategories.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const categories = [
+  { id: '1', title: 'Food' },
+  { id: '2', title: 'Handmade' },
+  { id: '3', title: 'Spices' },
+  { id: '5', title: 'clothing' },
+  { id: '4', title: 'Collectibles' },
+  { id: '6', title: 'Evertything' },
+]
+
+const Button: Story<CategoryListProps> = (args) => <TopCategories {...args} />
```
Comment (andrzej-onja) [inline src/components/TopCategories/TopCategories.stories.tsx]: Button ????

File: cypress/integration/TopCategories.spec.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        cy.contains('TopCategories')
+        cy.visit(
+            'http://localhost:6006/iframe.html?id=components-topcategories--test&args=&viewMode=story'
+        )
+        cy.contains('Top categories')
```
Comment (andrzej-onja) [inline cypress/integration/TopCategories.spec.js:14]: please also check if some or all buttons are present

Comment (andrzej-onja) [reply cypress/integration/TopCategories.spec.js:14]: { id: '1', title: 'Food' },
  { id: '2', title: 'Handmade' },
  { id: '3', title: 'Spices' },
  { id: '5', title: 'clothing' },
  { id: '4', title: 'Collectibles' },
  { id: '6', title: 'Evertything' },

**Review by tsipoy — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**